test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -1194,7 +1215,7 @@ describe('emitConversationEvent()', () => {
     });
 
     expectedEvent.removed.push(conversationState);
-    handler.emitConversationEvent('removed', conversationState, undefined);
+    handler.emitConversationEvent('removed', conversationState, undefined as any);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    handler.emitConversationEvent('removed', conversationState, undefined as unknown);
```